fladle-plugin/src/main/java/com/osacky/flank/gradle/FladlePluginDelegate.kt

```diff
@@ -106,6 +108,15 @@ class FladlePluginDelegate {
         environment(mapOf("GOOGLE_APPLICATION_CREDENTIALS" to config.serviceAccountCredentials.get()))
       }
       dependsOn(writeConfigProps)
+      if (config.dependOnAssemble.isPresent && config.dependOnAssemble.get()) {
+        val testedExtension = project.extensions.findByType(TestedExtension::class.java)
+        testedExtension?.testVariants?.configureEach {
+          if (testedVariant.assembleProvider.isPresent) {
+            dependsOn(testedVariant.assembleProvider.get())
```

Comment:
nice, thanks for finding and fixing this!

Reply:
Happy to help! 😃